### ISSUE TITLE
Changing Speed/Tempo Possible?

### ISSUE THREAD

**Issue by BartSar** [jrising/pysoundtouch#7]
Is it possible to change the tempo of the audio file with this library without changing pitch?

**Comment by jrising** [comment]
Yes, the `set_tempo(fraction)` can do this. I have not done this recently, so I believe the values of `fraction` can be between -0.95 (-95%) to 50 (5000%).
